Yael/apm network tab (#8632)
DataDog/documentation: content/en/tracing/visualization/trace.md

[1]: https://docs.datadoghq.com/infrastructure/process/?tab=linuxwindows
[2]: https://docs.datadoghq.com/infrastructure/process/?tab=linuxwindows#installation
{{% /tab %}}

{{% tab "Network" %}}

> Click on a service’s span to see network dependencies of the service making the request. Use key network performance metrics such as volume, errors (TCP retransmits) and network latency (TCP round-trip time) to differentiate between application-specific and network-wide issues, especially when no code errors have been generated. For instance, you can use network telemetry to determine whether high request latency is due to traffic overloading of the relevant application, or faulty dependencies with a downstream pod, security group, or any other tagged endpoint. Clicking on a process will bring you to the [Network Overview][1]. To view span-specific processes, [enable Network Performance Monitoring][2]. Related network telemetry is not currently supported for serverless traces.

Reviewer: ```suggestion
Click on a service’s span to see network dependencies of the service making the request. Use key network performance metrics such as volume, errors (TCP retransmits), and network latency (TCP round-trip time) to differentiate between application-specific and network-wide issues, especially when no code errors have been generated. For instance, you can use network telemetry to determine if high request latency is due to traffic overloading of the relevant application, or faulty dependencies with a downstream pod, security group, or any other tagged endpoint. Clicking on a process brings you to the [Network Overview][1]. To view span-specific processes, enable [Network Performance Monitoring][2].

**Note**: Related network telemetry is not currently supported for serverless traces.
```

> {{< img src="tracing/visualization/trace/trace_networks.png" alt="Trace Network Dependencies"  style="width:90%;">}}

Reviewer: ```suggestion
{{< img src="tracing/visualization/trace/trace_networks.png" alt="Trace Network Dependencies" style="width:90%;">}}
```
